pkg/grpc/dapr/dapr_api.go

```diff
@@ -164,18 +165,29 @@ func (d *daprGrpcAPI) InvokeBinding(ctx context.Context, in *runtime.InvokeBindi
 // NewDaprAPI_Alpha construct a grpc_api.GrpcAPI which implements DaprServer.
 // Currently it only support Dapr's InvokeService and InvokeBinding API.
 // Note: this feature is still in Alpha state and we don't recommend that you use it in your production environment.
-func NewDaprAPI_Alpha(ac *grpc_api.ApplicationContext) grpc_api.GrpcAPI {
+func NewDaprAPI_Alpha(
+	appId string,
+	hellos map[string]hello.HelloService,
+	configStores map[string]configstores.Store,
+	rpcs map[string]rpc.Invoker,
+	pubSubs map[string]pubsub.PubSub,
+	stateStores map[string]state.Store,
+	files map[string]file.File,
+	lockStores map[string]lock.LockStore,
+	sequencers map[string]sequencer.Store,
+	sendToOutputBindingFn func(name string, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error),
+	secretstores map[string]secretstores.SecretStore,
+) grpc_api.GrpcAPI {
 	// filter out transactionalStateStores
 	transactionalStateStores := map[string]state.TransactionalStore{}
-	for key, store := range ac.StateStores {
+	for key, store := range stateStores {
```

Comment:
same as above. The latest code was overwritten by mistake when resolving git conflicts 😢